packages/matchbox/src/components/Spinner/styles.js

```diff
@@ -0,0 +1,87 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { keyframes, css } from 'styled-components';
+
+export const circleOuter = props => {
+  let size = '60px';
+
+  if (props.size === 'small') {
+    size = '28px';
```

Comment:
I think small needs to be a bit smaller to fit inside small buttons, I also tried putting it into textfield as a prefix/suffix. Could also maybe make this a default size and just add another smaller one

```js
<Button size="small">
      <Spinner size="small" label="loading" />
    </Button>
<TextField prefix={<Spinner size="small" label="loading" />} />
```